.github/workflows/test.yml

```diff
@@ -98,15 +104,15 @@ jobs:
         run: wp-cli core install --url=127.0.0.1 --title=ConvertKit --admin_user=admin --admin_password=password --admin_email=wordpress@convertkit.local
 
       # env.INSTALL_PLUGINS is a list of Plugin slugs, space separated e.g. contact-form-7 woocommerce.
-      - name: Install and Activate Free Third Party WordPress Plugins
```

Comment:
I am curious about this change.

Reply:
Activation was a pointless step here, because Codeception later imports the skeleton database before running tests.  This overwrites which Plugins are active, as a Plugin's active/inactive state is stored in the database.

In this case, our skeleton database has no Plugins active, which is by design as each individual test activates the Plugin(s) needed to run that specific test in isolation, and deactivates them after.